### ISSUE TITLE
About Concept Community mining in ccig.py file

### ISSUE THREAD

**Issue by MollyShuu** [lancopku/Graph-to-seq-comment-generation#12]
Hello, _I would like to ask what is the purpose of **Concept Community mining in ccig.py file?**_
I did not find any content about community in the article,
_Is the construction of Graph related to the Concept Community? Is concept community the keywords in paper?_

**Comment by liweitj47** [comment]
No, they are not used in our method发自我的荣耀手机-------- 原始邮件 --------发件人： MollyShuu <notifications@github.com>日期： 2021年2月27日周六 傍晚6:52收件人： lancopku/Graph-to-seq-comment-generation <Graph-to-seq-comment-generation@noreply.github.com>抄送： Subscribed <subscribed@noreply.github.com>主    题： [lancopku/Graph-to-seq-comment-generation] About Concept Community mining in ccig.py file (#12)
Hello, I would like to ask what is the purpose of Concept Community mining in ccig.py file?
I did not find any content about community in the article,
Is the construction of Graph related to the Concept Community? Is concept community the keywords in paper?
